packages/arb-token-bridge-ui/src/util/NetworkUtils.ts

```diff
@@ -0,0 +1,95 @@
+import { L1Network, L2Network } from '@arbitrum/sdk'
+import { hexValue } from 'ethers/lib/utils'
+import { ChainId, getNetworkName, rpcURLs } from './networks'
+import { BigNumber } from 'ethers'
+import { Web3Provider } from '@ethersproject/providers'
+import EthereumLogo from '../assets/EthereumLogo.png'
+import ArbitrumOneLogo from '../assets/ArbitrumOneLogo.svg'
+import ArbitrumNovaLogo from '../assets/ArbitrumNovaLogo.png'
+
+/* 
+  The list which will be available for network selection in navbar-dropdowns
+ */
+export const networkSelectionList = [
+  ChainId.ArbitrumOne,
+  ChainId.ArbitrumNova,
+  ChainId.Mainnet
+]
+
+/* 
+  Icons and colors attached with our networks available for selection
+*/
+export const networkStyleMap: {
+  [chainId: number]: { img: string; btnThemeClass: string }
+} = {
+  [ChainId.Mainnet]: {
+    img: EthereumLogo,
+    btnThemeClass: 'bg-blue-arbitrum'
+  },
+  [ChainId.ArbitrumOne]: {
+    img: ArbitrumOneLogo,
+    btnThemeClass: 'bg-blue-arbitrum'
+  },
+  [ChainId.ArbitrumNova]: {
+    img: ArbitrumNovaLogo,
+    btnThemeClass: 'bg-brick-dark'
+  }
+}
+
+/*
+  - A very basic variant of `changeNetwork` function present in App.tsx (which initializes after the app load)
+  - The original App-injected version requires a network connection already
+  - This lightweight function doesn't require a connected network - simply switches the metamask network
+  - But it doesn't support very rich business logic as well, like showing custom alerts if user is connected to `Arbitrum` or not
+*/
+export const changeNetworkBasic = async (
```

Comment:
This method is highly similar to this one in `App.tsx`
Can we abstract them?

https://github.com/OffchainLabs/arb-token-bridge/blob/ef54393aecbe8316b02d2b665f562541eca96f23/packages/arb-token-bridge-ui/src/components/App/App.tsx#L263-L324